sdk/nodejs/src/types.ts

```diff
@@ -160,6 +160,10 @@ export interface DVCEvent {
 export interface DVCFeature {
     readonly _id: string
 
+    readonly _variation: string
```

Comment:
are there any unit tests we need to update as well?

Reply:
all the existing unit tests for allFeatures just pass in a test config and then check that what's returned matches testConfig.features, so there's nothing explicitly checking the fields on the features themselves